### YamlMime:ManagedReference
items:
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException
  id: EntityDataSourceValidationException
  children:
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  langs:
  - csharp
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException
  type: Class
  summary: "Fouten die optreden bij het valideren van de eigenschappen van een dynamische gegevensbron vertegenwoordigt."
  syntax:
    content: 'public sealed class EntityDataSourceValidationException : Exception, System.Web.DynamicData.IDynamicValidatorException'
  inheritance:
  - System.Exception
  implements:
  - System.Web.DynamicData.IDynamicValidatorException
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  id: '#ctor'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Maakt een nieuw exemplaar van de <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> klasse met een systeem geleverd bericht met een beschrijving van de fout."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van het nieuwe exemplaar voor een systeem geleverd bericht met een beschrijving van de fout, zoals &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot;.</xref:System.Exception.Message%2A> Dit bericht wordt rekening gehouden met de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException ();
    parameters: []
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Maakt een nieuw exemplaar van de <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> klasse met een opgegeven bericht met een beschrijving van de fout."
  remarks: "De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht opgegeven in `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het bericht dat de uitzondering wordt beschreven. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Maakt een nieuw exemplaar van de <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor of `null` als de <xref:System.Exception.InnerException%2A>eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht opgegeven in `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het bericht dat de uitzondering wordt beschreven. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur."
    - id: innerException
      type: System.Exception
      description: "De uitzondering die de oorzaak van de huidige uitzondering. Als de `innerException` parameter is niet <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  id: System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  isEii: true
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  type: Property
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Hiermee haalt u de uitzonderingen die optreden wanneer een nieuw of bewerkt gegevensveld wordt gevalideerd."
  remarks: "Dit lid is een expliciete interface lid-implementatie. Het kan worden gebruikt alleen wanneer de <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>exemplaar is geconverteerd naar een <xref:System.Web.DynamicData.IDynamicValidatorException>interface.</xref:System.Web.DynamicData.IDynamicValidatorException> </xref:System.Web.UI.WebControls.EntityDataSourceValidationException>"
  syntax:
    content: System.Collections.Generic.IDictionary<string,Exception> System.Web.DynamicData.IDynamicValidatorException.InnerExceptions { get; }
    return:
      type: System.Collections.Generic.IDictionary{System.String,System.Exception}
      description: "Een verzameling waartoe de uitzonderingen die zijn opgetreden."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
- uid: System.Collections.Generic.IDictionary{System.String,System.Exception}
  parent: System.Collections.Generic
  isExternal: true
  name: IDictionary<String,Exception>
  nameWithType: IDictionary<String,Exception>
  fullName: System.Collections.Generic.IDictionary<System.String,System.Exception>
  spec.csharp:
  - uid: System.Collections.Generic.IDictionary`2
    name: IDictionary
    nameWithType: IDictionary
    fullName: IDictionary<System.String,System.Exception>
  - name: <
    nameWithType: <
    fullName: <
  - uid: System.String
    name: String
    nameWithType: String
    fullName: String
  - name: ','
    nameWithType: ','
    fullName: ','
  - uid: System.Exception
    name: Exception
    nameWithType: Exception
    fullName: Exception
  - name: '>'
    nameWithType: '>'
    fullName: '>'
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
